inference-engine/src/mkldnn_plugin/mkldnn_node.h

```diff
@@ -651,6 +589,7 @@ class MKLDNNNode {
 protected:
     bool canBePerformedAsScaleShift(const MKLDNNNode *parentNode = nullptr) const;
     bool canFuseSimpleOperation(const MKLDNNNodePtr& node) const;
+    void fillScalesAndShifts(const MKLDNNNode *parentNode, std::vector<float> &scales, std::vector<float> &shifts, const int align = -1);
```

Comment:
Not sure I like the implementation. For now we can keep as is, but in general I would prefer it to be placed outside of the node API.

Reply:
todo was added